uber/common.py

```diff
@@ -5,6 +5,7 @@
 import json
 import math
 import html
+import uuid
```

Comment:
We already import a `uuid` below:

`from sideboard.lib.sa import declarative_base, SessionManager, UTCDateTime, UUID, CoerceUTF8 as UnicodeText`